guides/common/attributes-satellite.adoc

```diff
@@ -111,9 +111,11 @@
 :smart-proxy-context: capsule
 :smart-proxy-context-titlecase: Capsule
 :smart-proxy-principal: {smart-proxy-context}
+:smartproxy-installer-package: satellite-capsule
 :SmartProxies: Capsules
 :smartproxy_port: 9090
 :smartproxy-example-com: capsule.example.com
+:smartproxy-installer-package: satellite-capsule
```

Comment:
```suggestion
```

Reply:
I noticed this too when I showed it during the meeting.